app/src/main/java/com/github/wnder/LoginActivity.java

```diff
@@ -0,0 +1,110 @@
+package com.github.wnder;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+import com.google.android.gms.auth.api.signin.GoogleSignIn;
+import com.google.android.gms.auth.api.signin.GoogleSignInAccount;
+import com.google.android.gms.auth.api.signin.GoogleSignInClient;
+import com.google.android.gms.auth.api.signin.GoogleSignInOptions;
+import com.google.android.gms.common.api.ApiException;
+import com.google.android.gms.tasks.Task;
+
+public class LoginActivity extends AppCompatActivity implements View.OnClickListener {
+
+    private View signInButton;
+    private TextView textLogin;
+    private GoogleSignInClient client;
+    private int RC_SIGN_IN = 10; //nombre arbitraire
```

Comment:
Should all comments be written in English?

Reply:
Yes they should, these were written by @leonardvaney I forgot to remove or translate them